meta/store.go

```diff
@@ -1664,6 +1678,27 @@ func (fsm *storeFSM) Apply(l *raft.Log) interface{} {
 	return err
 }
 
+func (fsm *storeFSM) applyRemovePeerCommand(cmd *internal.Command) interface{} {
+	ext, _ := proto.GetExtension(cmd, internal.E_RemovePeerCommand_Command)
+	v := ext.(*internal.RemovePeerCommand)
+
+	id := v.GetID()
+	addr := v.GetAddr()
+	//Remove that node from the peer
+	if id == fsm.id {
```

Comment:
Is this right?  This looks like the `RemovePeerCommand` will only remove itself?